rhv4/README.md

```diff
@@ -0,0 +1,5 @@
+# Red Hat Virtualization 4
+
+As oVirt moves to be based on el8, the contents of `rhv4` will also move to be based on el8.
+
+If you need content for oVirt based on el7, use the Red Hat Enterprise Linux 7 (rhel7) content.
```

Comment:
My understanding was that oVirt is the open-source project and Red Hat Virtualization 4 is based on top of that. This content is for Red Hat Virtualization. oVirt isn't mentioned anywhere, so I would prefer to not mention it in the README.

Reply:
Thanks for your feedback, updated.